wrappers/node/test/suite1/ariesvcx-credential.test.ts

```diff
@@ -94,9 +94,9 @@ describe('Credential:', () => {
       const connection = await createConnectionInviterRequested();
       const data = await dataCredentialCreateWithOffer();
       const credential = await Credential.create(data);
-      assert.equal(await credential.getState(), StateType.RequestReceived);
+      assert.equal(await credential.getState(), 0);
```

Comment:
Should use the TS enums rather than numeric values

Reply:
Did that everywhere in wrapper unit tests